pandas/core/indexes/base.py

```diff
@@ -1644,21 +1647,184 @@ def is_unique(self) -> bool:
 
     @property
     def has_duplicates(self) -> bool:
+        """
+        Check if the Index has duplicate values.
+
+        Returns
+        -------
+        boolean
```

Comment:
```suggestion
        bool
```
https://pandas.io/docs/development/contributing_docstring.html#parameter-types